index.html

```diff
@@ -28,5 +29,126 @@ <h1>I’m Mariane Glad to see you!</h1>
             <li><a href="#"><img src="images/angellist.png" alt=""></a></li>
         </ul>
     </section>
+    <section class="app-works">
+        <div class="project-card container-v">
+            <img src="images/snapshots/Snapshoot Portfolio.svg" alt="Snapshot of project">
+            <h3 class="project-title">Tonic</h3>
+            <div class="details container-h">
+                <span class="client">CANOPY</span>
+                <img class="dot-figure" src="images/Counter.svg">
+                <span class="role">Back End Dev</span>
+                <img class="dot-figure" src="images/Counter.svg">
+                <span class="year">2015</span>
+            </div>
+            <p class="simple-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="container-h">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button class="complex-button">See Project</button>
+        </div>
+        <div class="project-card container-v">
+            <img src="images/snapshots/Snapshoot Portfolio (1).svg" alt="Snapshot of project">
+            <h3 class="project-title">Multi-Post<br>Stories</h3>
+            <div class="details container-h">
+                <span class="client">CANOPY</span>
+                <img class="dot-figure" src="images/Counter.svg">
+                <span class="role">Back End Dev</span>
+                <img class="dot-figure" src="images/Counter.svg">
+                <span class="year">2015</span>
+            </div>
+            <p class="simple-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="container-h">
+                <li>html</li>
```

Comment:
- [ ] Please use an indentation of two spaces in all your documents for readability. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).